sdk/state/close.go

```diff
@@ -116,5 +116,6 @@ func (c *Channel) makeCloseTx(observationPeriodTime time.Duration, observationPe
 		IterationNumber:            c.latestCloseAgreement.IterationNumber,
 		AmountToInitiator:          c.initiatorClaimAmount(),
 		AmountToResponder:          c.responderClaimAmount(),
+		Asset:                      c.latestCloseAgreement.Balance.Asset,
```

Comment:
@nikhilsaraf fyi I actually needed to include this other change so the test doesn't break, sorry for adding late. Because the closing transaction created in state/close.go needs the asset now, we're pulling it from the `latestCloseAgreement`. Similarly how the amount is pulled from the `latestCloseAgreement`

going to merge now but I can make changes in next PR if needed

Reply:
thanks for highlighting.

One thought is that the asset should probably live on the Channel rather than pulling from the latestCloseAgreement, but we don't need to worry about that since that's merely a design/interface optimization that we can improve later if we feel the need. I'm ok with leaving as-is. 

Thanks!